source/mesh_deformation/fastscape.cc

```diff
@@ -1898,7 +1934,19 @@ namespace aspect
 
           prm.enter_subsection("Marine parameters");
           {
-            sea_level = prm.get_double("Sea level");
+            use_sea_level_function = prm.get_bool("Use sea level function");
+            if (use_sea_level_function)
+            {
+              prm.enter_subsection("Sea level function");
+              {
+                sea_level_function.parse_parameters(prm);
+              }
+              prm.leave_subsection();
+            } 
+            else
+            {
+              sea_level_constant_value = prm.get_double("Sea level");
+            }
```

Comment:
Like in the other PR, I think it would be useful to also "poison" the `sea_level_constant_value` if you're in the `if` branch:
```suggestion
            use_sea_level_function = prm.get_bool("Use sea level function");
            if (use_sea_level_function)
            {
              prm.enter_subsection("Sea level function");
              {
                sea_level_function.parse_parameters(prm);
              }
              prm.leave_subsection();
              
              // If using the function description for the sea level, no parts of the code
              // base should use the sea_level_constant_value variable. Poison it to 
              // make sure it really isn't used anywhere:
              sea_level_constant_value = numbers::signaling_nan<double>();
            } 
            else
            {
              sea_level_constant_value = prm.get_double("Sea level");
            }
```